transform/transformer_test.go

```diff
@@ -379,23 +380,31 @@ func TestSaveValue(t *testing.T) {
 }
 
 func TestTransformer(t *testing.T) {
+	const parallelRuns = 4
 	for _, test := range transformerTests {
 		tr, err := NewTransformer(test.schema, test.transformIdentifier)
 		if err != nil {
 			t.Fatalf("Test %q - failed to initialize transformer: %v", test.description, err)
 		}
+		testFunc := func(description string, in json.RawMessage, wantErr bool, want json.RawMessage) func(t *testing.T) {
+			return func(t *testing.T) {
+				t.Parallel()
+				got, err := tr.Transform(in)
 
-		got, err := tr.Transform(test.in)
-
-		switch {
-		case test.wantErr && err != nil:
-			continue
-		case test.wantErr && err == nil:
-			t.Errorf("Test %q - got nil, want error", test.description)
-		case !test.wantErr && err != nil:
-			t.Errorf("Test %q - got error, want nil: %v", test.description, err)
-		case !reflect.DeepEqual(got, test.want):
-			t.Errorf("Test %q - got\n%s\nwant\n%s", test.description, got, test.want)
+				switch {
+				case wantErr && err != nil:
+					return
+				case wantErr && err == nil:
+					t.Errorf("Test %q - got nil, want error", description)
+				case !test.wantErr && err != nil:
```

Comment:
should this also be wantErr and not test.wantErr?

Reply:
good catch, updating